third_party/istio-latest/install-istio.sh

```diff
@@ -29,6 +29,20 @@ tar xzf ${ISTIO_TARBALL}
 # Install Istio
 ./istio-${ISTIO_VERSION}/bin/istioctl install -f "$(dirname $0)/$1"
 
+# Enable mTLS STRICT
+if [[ -n "$KIND" ]]; then
+  cat <<EOF | kubectl apply -f -
```

Comment:
I personally prefer to putting a yaml file and run `kubectl apply -f "$(dirname $0)/extra/global-mtls.yaml"`. But either is fine.

Reply:
+1 to moving yaml outside of `bash`. already easy to make mistakes with bash without embedding :D